text/0000-input-macros.md

```diff
@@ -0,0 +1,484 @@
+- Feature Name: input_macros
+- Start Date: 2025-04-19
+- RFC PR: [rust-lang/rfcs#3799](https://github.com/rust-lang/rfcs/pull/3799)
+- Rust Issue: [rust-lang/rust#0000](https://github.com/rust-lang/rust/issues/0000)
+
+# Summary
+[summary]: #summary
+
+This RFC proposes the addition of macros or some functions that can be used to 
+read input from the user in a more ergonomic way, similar to the 
+[Input built-in function in Python](https://peps.python.org/pep-3111/). 
+
+With this initiative, we can build a small interactive programs that reads input 
+from standard input and writes output to standard output. This is 
+well-established as a simple and fun way of learning and teaching Rust as a 
+new programming language. 
+
+```rust
+println!("Please enter your name: ");
```

Comment:
Did you mean `print` rather than `println`?

```suggestion
print!("Please enter your name: ");
```

Reply:
Thanks